modules/pod-short-term-auth-gcp-deploy-pod.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-short-term-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[discrete]
+[id="pod-short-term-auth-gcp-deploy-pod_{context}"]
+= Deploying customer workloads that authenticate with {gcp-wid-short}
+
+To use short-term authentication in your application, you must configure its related pods to use the {product-title} service account.
+Using the {product-title} service account triggers the webhook to mutate the pods so they can impersonate the {gcp-short} service account.
+
+The following example of demonstrates how to deploy a pod that uses the {product-title} service account and verify the configuration.
```

Comment:
> The following example of demonstrates 

"The following example demonstrates" instead?

Reply:
Great catch, thanks!